go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/onsi/gomega v1.20.2
 	github.com/pkg/errors v0.9.1
 	github.com/psampaz/go-mod-outdated v0.8.0
+	github.com/submariner-io/admiral v0.13.0
```

Comment:
**shipyard** cannot depend on **admiral** - that would create a circular dependency.

Reply:
Got it.
Will remove.

Reply:
Done